### DISCUSSION TITLE
Quickstart in Windows

### DISCUSSION THREAD

**Discussion by jengalbert2022** [wgu-opensource/osmt#145] [General]
I'm on Windows 11 and I'm trying out the Quickstart feature, using the directions under the How-To. I'm stuck on creating a Local OSMT. I'm lost on how to use the OSMT CLI utility in Windows. CMD appears to either not recognize it as a file or if I double click to launch it from file manager it exits the program on its own after one second from launching it. The step I'm stuck on is the "Initialize your environment files by running this command : ./osmt_cli.sh -i", as the shell script won't launch for me. Also, I've already installed Git and Docker.

**Comment by sleeman1** [comment]
For windows, you will want to run the script using Git Bash. 

-Chad

**Reply by jengalbert2022** [reply]
I tried running the command in Git Bash having cd into the directory but it says no file is found

**Reply by sleeman1** [reply]
You cloned the repo, you cd /osmt and when you ls you don't see the script/files listed?  It sounds like maybe the repo was not cloned or the location may be different?
![image](https://user-images.githubusercontent.com/64618524/152252091-2cc2e739-a495-4651-b698-2344498688fa.png)

**Reply by JohnKallies** [reply]
@jengalbert2022 Try verifying that you are using the latest changes in develop? Make sure you have this commit in play?

`231fb70 (origin/develop, origin/HEAD) Update Quickstart to use osmt-app Docker image (#136)`

Can you please reply with the top few lines from this command's output?
`git log --pretty=oneline`

Thank you

**Reply by jengalbert2022** [reply]
I did clone the correct repo and see those files, as well as have that commit. I've decided to use a Mac instead of a Windows machine so it works for me now,

**Comment by JohnKallies** [comment]
@jengalbert2022, please also see this... It may or may not be related to your problem...
* https://github.com/wgu-opensource/osmt/discussions/146
